### PR TITLE
Create Ciname Hall controller

### PR DESCRIPTION
**Pull request by mateuszCabala95**

### CODE REVIEW

**Review by jkopyto — CHANGES_REQUESTED** [review]

File: src/index.ts [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import App from "./App"
 import AuthenticationController from "./routes/controllers/authentication.controller"
+import CinemaHall from "./routes/controllers/cinemaHall.controller"
```
Comment (DaveTheWebDev) [inline src/index.ts:3]: Consider changing CinemaHall to CinemaHallController to keep the schema.

**Review by DaveTheWebDev — APPROVED** [review]
Everything looks good, if you are using only one parameter in function, you can omit the brackets.

File: src/routes/dtos/CinemaHall.dto.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { IsNumber, IsString } from "class-validator"
+
+export default class CinemaHallDTO {
+  @IsString()
+  name: string
+
+  @IsNumber()
```
Comment (kgutka) [inline src/routes/dtos/CinemaHall.dto.ts:7]: można jeszcze dodać `@Min(1)`

Comment (jkopyto) [reply src/routes/dtos/CinemaHall.dto.ts:7]:
```
@Min(3)
@Max(32)
```

**Pull request merged**